packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -37,6 +40,14 @@ interface IButtonClassNames {
 
 export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState> implements IButton {
 
+  private get _splitButton(): boolean {
+    return (this.props.menuProps != null && this.props.onClick != null);
```

Comment:
The issue was that it wouldn't return the same thing with `undefinied`. This is now fixed with not not !! (although kind of ugly...). Thanks!